examples/seismic/accuracy.ipynb

```diff
@@ -0,0 +1,415 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from examples.seismic.acoustic import AcousticWaveSolver\n",
+    "from examples.seismic import Model, RickerSource, Receiver\n",
+    "from devito import clear_cache, set_log_level\n",
+    "from opescibench import LinePlotter\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "from matplotlib import cm"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Model with fixed time step value\n",
+    "class ModelBench(Model):\n",
```

Comment:
I find it clearer that way, I also don't really like overwriting the CFL in the normal `Model`  which would lead to people doing stupid shit  (like allways putting 4ms because that's what's in the shot record) and complain it doesn't work.

Reply:
\<personal opinion> Trying to prevent people from doing "stupid shit" is a futile endeavour as the persistence of idiocy knows no bounds.<\personal opinion> :wink:

But anyway, this might just come down to a matter of taste. I'm personally not a fan because I think this particular use case demonstrates that there is sufficient reason for an override, but I'm happy for it to go in as is.